### PR TITLE
Update Azure links

### PR DESCRIPTION
**Pull request by guardrex**
Addresses Part 4 of #2580

### CODE REVIEW

File: docs/standard/microservices-architecture/architect-microservice-container-applications/containerize-monolithic-applications.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ From an infrastructure perspective, each server can run many applications within
 
 **Figure 4-2**. Monolithic approach: Host running multiple apps, each app running as a container
 
-Monolithic applications in Microsoft Azure can be deployed using dedicated VMs for each instance. Additionally, using [Azure VM Scale Sets](https://azure.microsoft.com/en-us/documentation/services/virtual-machine-scale-sets/), you can easily scale the VMs. [Azure App Service](https://azure.microsoft.com/en-us/services/app-service/) can also run monolithic applications and easily scale instances without requiring you to manage the VMs. Since 2016, Azure App Services can run single instances of Docker containers as well, simplifying deployment.
+Monolithic applications in Microsoft Azure can be deployed using dedicated VMs for each instance. Additionally, using [Azure VM Scale Sets](https://azure.microsoft.com/documentation/services/virtual-machine-scale-sets/), you can easily scale the VMs. [Azure App Service](https://azure.microsoft.com/services/app-service/) can also run monolithic applications and easily scale instances without requiring you to manage the VMs. Since 2016, Azure App Services can run single instances of Docker containers as well, simplifying deployment.
```
Comment (mairaw) [inline docs/standard/microservices-architecture/architect-microservice-container-applications/containerize-monolithic-applications.md]: change https://azure.microsoft.com/documentation/services/virtual-machine-scale-sets/ to https://docs.microsoft.com/azure/virtual-machine-scale-sets/

File: docs/standard/microservices-architecture/architect-microservice-container-applications/docker-application-state-data.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ The following solutions are used to manage persistent data in Docker application
 
 -   Remote data sources like SQL or NoSQL databases, or cache services like [Redis](https://redis.io/).
 
--   [Azure Storage](https://azure.microsoft.com/en-us/documentation/services/storage/), which provides geo-distributable storage, providing a good long-term persistence solution for containers.
+-   [Azure Storage](https://azure.microsoft.com/documentation/services/storage/), which provides geo-distributable storage, providing a good long-term persistence solution for containers.
```
Comment (mairaw) [inline docs/standard/microservices-architecture/architect-microservice-container-applications/docker-application-state-data.md]: change https://azure.microsoft.com/documentation/services/storage/ to https://docs.microsoft.com/azure/storage/

File: docs/standard/microservices-architecture/architect-microservice-container-applications/index.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Enterprise applications can be complex and are often composed of multiple servic
...
 
-When you design a container image, you will see an [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/) definition in the Dockerfile. This defines the process whose lifetime controls the lifetime of the container. When the process completes, the container lifecycle ends. Containers might represent long-running processes like web servers, but can also represent short-lived processes like batch jobs, which formerly might have been implemented as Azure [WebJobs](https://azure.microsoft.com/en-us/documentation/articles/websites-webjobs-resources/).
+When you design a container image, you will see an [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/) definition in the Dockerfile. This defines the process whose lifetime controls the lifetime of the container. When the process completes, the container lifecycle ends. Containers might represent long-running processes like web servers, but can also represent short-lived processes like batch jobs, which formerly might have been implemented as Azure [WebJobs](https://azure.microsoft.com/documentation/articles/websites-webjobs-resources/).
```
Comment (mairaw) [inline docs/standard/microservices-architecture/architect-microservice-container-applications/index.md]: change https://azure.microsoft.com/documentation/articles/websites-webjobs-resources/ to https://docs.microsoft.com/azure/app-service-web/websites-webjobs-resources

File: docs/standard/microservices-architecture/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md [hunk 4]
```diff
@@ -64,17 +64,17 @@ Azure Service Fabric
...
 
-> [Service Fabric](https://azure.microsoft.com/en-us/documentation/articles/service-fabric-overview/) is a Microsoft microservices platform for building applications. It is an [orchestrator](https://azure.microsoft.com/en-us/documentation/articles/service-fabric-cluster-resource-manager-introduction/) of services and creates clusters of machines. By default, Service Fabric deploys and activates services as processes, but Service Fabric can deploy services in Docker container images. More importantly, you can mix services in processes with services in containers in the same application.
+> [Service Fabric](https://azure.microsoft.com/documentation/articles/service-fabric-overview/) is a Microsoft microservices platform for building applications. It is an [orchestrator](https://azure.microsoft.com/documentation/articles/service-fabric-cluster-resource-manager-introduction/) of services and creates clusters of machines. By default, Service Fabric deploys and activates services as processes, but Service Fabric can deploy services in Docker container images. More importantly, you can mix services in processes with services in containers in the same application.
```
Comment (mairaw) [inline docs/standard/microservices-architecture/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md]: - change https://azure.microsoft.com/documentation/articles/service-fabric-overview/ to https://docs.microsoft.com/azure/service-fabric/service-fabric-overview
-  change https://azure.microsoft.com/documentation/articles/service-fabric-cluster-resource-manager-introduction/ to https://docs.microsoft.com/azure/service-fabric/service-fabric-cluster-resource-manager-introduction

File: docs/standard/microservices-architecture/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md [hunk 5]
```diff
@@ -64,17 +64,17 @@ Azure Service Fabric
...
+> Service Fabric services can be developed in many ways, from using the [Service Fabric programming models ](https://azure.microsoft.com/documentation/articles/service-fabric-choose-framework/)to deploying [guest executables](https://azure.microsoft.com/documentation/articles/service-fabric-deploy-existing-app/) as well as containers. Service Fabric supports prescriptive application models like [stateful services](https://azure.microsoft.com/documentation/articles/service-fabric-reliable-services-introduction/) and [Reliable Actors](https://azure.microsoft.com/documentation/articles/service-fabric-reliable-actors-introduction/).
```
Comment (mairaw) [inline docs/standard/microservices-architecture/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md]: review the remainder https://azure.microsoft.com/documentation/ links to go to the new docs.microsoft.com link without en-us

**Review by mairaw — CHANGES_REQUESTED** [review]
LGTM but I'd change the URLs from https://azure.microsoft.com/documentation/ to the new docs.microsoft.com URL. I've commented in a few cases, not all, but it should be easy to find them.

**Comment by GuardRex** [comment]
@mairaw Ah ... yes ... np. These were all handled by script, and another little script will do the job. I'll take care of it. :cake:

**Comment by GuardRex** [comment]
@mairaw Script only found four lines in four files with this. Did I catch all of them looking for ...

```
azure.microsoft.com/documentation/services
```

**Comment by mairaw** [comment]
There are still a couple left such as https://azure.microsoft.com/documentation/articles/websites-webjobs-resources/

**Comment by GuardRex** [comment]
I see. I only checked with a "services" segment ... I didn't address an "articles" segment. I'll get that on the next pass.

**Comment by GuardRex** [comment]
@mairaw I've added relative links as an action item on the main issue: https://github.com/dotnet/docs/issues/2580. I'll do those on a new PR after these go in.

**Comment by mairaw** [comment]
@GuardRex do you mean changing the docs.microsoft.com links to /azure? If so, these are ok like this because they're part of the microservices book. if we change them to relative links, then they can't use that as a source for a new edition as-is.

**Review by mairaw — APPROVED** [review]

**Pull request merged**